### PR TITLE
Add Nokia here maps

### PR DESCRIPTION
**Pull request by kozer**
Nokia HERE maps added.Add L.Leaflet.Provider layer as bower-dev depedency.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "leaflet-tilelayer-geojson": "*",
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "Leaflet.awesome-markers": "*",
+    "leaflet-providers": "*",
```
Comment (nmccready) [inline bower.json]: Is there a specific version avail?

Comment (kozer) [inline bower.json]: Yes the latest is "version": "1.1.5"

Comment (nmccready) [inline bower.json]: Cool then please make it "^1.1.5" or "~1.1.5"

**Comment by nmccready** [comment]
LGTM

**Pull request merged**